eZ/Publish/Core/LocationReference/ConfigResolver/LocationConfigResolverInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace eZ\Publish\Core\LocationReference\ConfigResolver;
+
+use eZ\Publish\Core\LocationReference\LocationReference;
+use eZ\Publish\API\Repository\Values\Content\Location;
+
+interface LocationConfigResolverInterface
```

Comment:
Is it internal?

Reply:
Well, technically it should be the part of the API, same as the `eZ\Publish\Core\MVC\ConfigResolverInterface` 😉

Reply:
> Well, technically it should be the part of the API, same as the `eZ\Publish\Core\MVC\ConfigResolverInterface`

So make it a part of the API, I mean the new interface only :P